app/controllers/schemes_controller.rb

```diff
@@ -103,6 +103,10 @@ def validation_errors(scheme_params)
         @scheme.errors.add(key.to_sym)
       end
     end
+
+    if @scheme.arrangement_type_same? && scheme_params[:arrangement_type] != "The same organisation that owns the housing stock"
```

Comment:
Note to self: 

If possible change to not refer to the text value (as this is likely to change)

Reply:
This seems like a good idea before merging still?

Reply:
So, the params come thru as strings and not the values we store. I think it might be a part of a bigger refactor, cause all of our values come thru like that atm

Reply:
For context, we discussed this with Dushan over the past few days.
While the label for an option can change reasonably often (design/year changes), the letter code/ID won't.

Reply:
@kosiakkatrina I think the best solution to that for now is something like:

```ruby
if Scheme::CONSTANT[scheme_params[:arrangement_type]] == "D"
```

etc. So still a string comparison really but at least you have a single source of truth for what that string is in the code base.

Reply:
Okay, had the same thought. Happy to update